frontend/src/routes/milestone/+page.svelte

```diff
@@ -2,23 +2,63 @@
 
```

Comment:
I'm not sure if there's a general rule - but this page in particular is only a temporary one that grabs some data to demo the component, it can be removed once the Milestone component is integrated into the main site